views/beneficial-owner-individual.html

```diff
@@ -11,65 +11,8 @@
-      {% include "includes/heading/bo-and-mo-individuals-heading.html" %}
-
-      <form method="post">
-
-        {% include "includes/inputs/fields/first-name-input.html" %}
-
-        {% set hintLastNameText = "This is also known as your family name." %}
-        {% include "includes/inputs/fields/last-name-input.html" %}
-
-        {% include "includes/inputs/date/date-of-birth-input.html" %}
-
-        {% include "includes/inputs/nationality/nationality.html" %}
-        {% include "includes/inputs/nationality/second-nationality.html" %}
-
-        {% set hintResidentialAddress = "The beneficial owner’s home address would be shown on the public register as the correspondence address." %}
-        {% include "includes/inputs/address/residential-address-input.html" %}
-
-        {% set dateText = "When did they become a beneficial owner for the overseas entity?" %}
-        {% include "includes/inputs/date/start-date-input.html" %}
-
-        {% call govukFieldset({
-          legend: {
-            text: "What is their nature of control?",
-            classes: "govuk-fieldset__legend--m",
-            isPageHeading: false
-          }
-        }) %}
-
-        {% set errorMessage = errors.beneficial_owner_nature_of_control_types if errors %}
-        {% include "includes/inputs/nature-of-control-checkboxes/beneficial-owner-nature-of-control-types.html" %}
-
-        {% set errorMessage = {} if errors.beneficial_owner_nature_of_control_types %}
-        {% include "includes/inputs/nature-of-control-checkboxes/trustees-nature-of-control-types.html" %}
-
-        {% set errorMessage = {} if errors.beneficial_owner_nature_of_control_types %}
-        {% include "includes/inputs/nature-of-control-checkboxes/non-legal-firm-nature-of-control-types.html" %}
-
-        {% endcall %}
-
-        {% set sanctionsListText = "Are they on the sanctions list?" %}
-        {% set sanctionsListHintText = "This means that they are subject to sanctions under the Sanctions and Anti-Money Laundering Act 2018." %}
-        {% include "includes/inputs/fields/is-on-sanctions-list-input.html" %}
-
-        {% set paragraphOne = '<p>We will not show the beneficial owner’s home address unless it is the same as their correspondence address. We’ll only show the month and year of their date of birth.</p>' %}
-        {% include "includes/inset-text.html" %}
-
-        {% include "includes/save-and-continue-button.html" %}
-
-        {% set removePathURL = OE_CONFIGS.BENEFICIAL_OWNER_INDIVIDUAL_URL + OE_CONFIGS.REMOVE + "/" + id %}
-        {% set nameToBeRemoved = first_name + " " + last_name %}
-        {% include "includes/remove-button.html" %}
-
-      </form>
-    </div>
-  </div>
+  {% include "includes/page-templates/beneficial-owner-individual.html" %}
+  {% set removePathURL = OE_CONFIGS.BENEFICIAL_OWNER_INDIVIDUAL_URL + OE_CONFIGS.REMOVE + "/" + id %}
+  {% set nameToBeRemoved = first_name + " " + last_name %}
+  {% include "includes/remove-button.html" %}
```

Comment:
the includes/page-templates/beneficial-owner-individual.html has a remove button included at the bottom of it, but this page (views/beneficial-owner-individual.html) also has a remove button.
looking at the original version of views/beneficial-owner-individual.html, i can only see the one remove button. Does this cause 2 remove buttons to appear in the registration journey?

Reply:
I've checked out the branch and it does show 2 remove buttons when changing an BO individual
<img width="927" alt="Screenshot 2023-02-17 at 15 49 43" src="https://user-images.githubusercontent.com/11330756/219701572-b076e6b6-9388-43a5-a5d1-02cd2c63362e.png">

Reply:
Looks ok now, thanks